config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  get 'static_pages/home'
+  get 'static_pages/help'
+  get 'static_pages/contact'
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  scope "(:locale)", locale: /en|vi/ do; end
```

Comment:
```suggestion
  scope "(:locale)", locale: /en|vi/ do
    
    get 'static_pages/home'
    get 'static_pages/help'
    get 'static_pages/contact'
  end
```

tự fix nháy đôi đi cu